config.go

```diff
@@ -608,6 +608,17 @@ func (s *Settings) CreateInstallationToken(ctx context.Context, owner, repo stri
 	return token.GetToken(), nil
 }
 
+// CreateInstallationTokenWithDefaultOwnerRepo returns an installation token when we do not care about
+// the owner/repo that we are calling the Github function with. (ie checking rate limit)
```

Comment:
tiniest grammar nit: `GitHub function with (i.e. checking rate limit).` (i.e. is an acronym, period should come after parenthesis, capitalize Hub)